Public/Src/Engine/UnitTests/Scheduler.IntegrationTest/BaselineTests.cs

```diff
@@ -961,6 +964,95 @@ public void SearchPathEnumerationTool()
             RunScheduler().AssertCacheHit(pip.PipId);
         }
 
+
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void AugmentedWeakFingerprint(bool augmentWeakFingerprint)

```

Comment:
>AugmentedWeakFingerprint [](start = 20, length = 24)

so this test covers the scenario where we are adding new augmented FPs under a common pathset. Should we have a test for scenario, where we are not growing the number of new FPs but growing the number of SFPs?